keyboards/gboards/k/faunchpad/info.json

```
@@ -0,0 +1,30 @@
+{
+    "keyboard_name": "FaunchPad",
+    "manufacturer": "g Heavy Industries",
+    "url": "",
+    "maintainer": "germ",
+    "usb": {
+        "vid": "0x1337",
+        "pid": "0x0006",
+        "device_version": "0.0.1"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "width": 4,
+    "height": 2,
+    "layouts": {
+      "LAYOUT_faunch": {
```

This change will need to be reflected in the keymaps as well.
```suggestion
      "LAYOUT": {
```